src/faldbt/lib.py

```diff
@@ -401,12 +410,32 @@ def _replace_relation(
                 profiles_dir,
                 six.text_type(create_stmt).strip(),
                 profile_target=profile_target,
+                adapter=adapter,
+            )
+            adapter.drop_relation(new_relation)
+        elif adapter.type() == "snowflake":
+            create_stmt = (
+                f"create or replace table {original_relation} clone {new_relation}"
+            )
+            _execute_sql(
+                project_dir,
+                profiles_dir,
+                six.text_type(create_stmt).strip(),
+                profile_target=profile_target,
+                adapter=adapter,
             )
             adapter.drop_relation(new_relation)
         else:
             adapter.rename_relation(new_relation, original_relation)
+
         adapter.connections.commit_if_has_connection()
 
+        config = parse.get_dbt_config(
+            project_dir, profiles_dir, profile_target=profile_target
+        )
+        manifest = parse.get_dbt_manifest(config)
+        adapter.set_relations_cache(manifest, True)
```

Comment:
What does this do?

Reply:
I will add a comment, it resets the cache.